### PR TITLE
CIS compliance: kv, alerts, sql audit

### LINKED ISSUE #190: Enable KV purge protection for prod
CIS 8.5 - enable purge protection for the key vault. Suggest to do this just for prod.

### PR DESCRIPTION
**Pull request by t-young31**
## Resolves #190 #188 #172

### CODE REVIEW

**Review by jjgriff93 — COMMENTED** [review]
Good stuff - happy to look again when not draft, just lemme know

File: config.sample.yaml [hunk 1]
```diff
@@ -27,3 +27,8 @@ serve:
   github_owner: __CHANGE_ME__
   github_app_id: __CHANGE_ME__
   github_app_installation_id: __CHANGE_ME__
+
+alert_recipeients:  # Optional
+  - firstperson:
```
[the remark this answers is not in the document]
Comment (t-young31) [reply config.sample.yaml]: For me that's less readable but happy to yield!

File: config.sample.yaml [hunk 2]
```diff
@@ -27,3 +27,8 @@ serve:
   github_owner: __CHANGE_ME__
   github_app_id: __CHANGE_ME__
   github_app_installation_id: __CHANGE_ME__
+
+alert_recipeients:  # Optional
```
[the remark this answers is not in the document]
Comment (t-young31) [reply config.sample.yaml]: hmm, yeah. is there definitely going to be other things in that block?

[the remark this answers is not in the document]
Comment (jjgriff93) [reply config.sample.yaml]: Happy to leave it until there is!

**Review by jjgriff93 — APPROVED** [review]
Lovely

File: config.sample.yaml [hunk 3]
```diff
@@ -27,3 +27,7 @@ serve:
   github_owner: __CHANGE_ME__
   github_app_id: __CHANGE_ME__
   github_app_installation_id: __CHANGE_ME__
+
+alert_recipients:  # Optional
+  - name: __CHANGE_ME__
+    email: __CHANGE_ME__@company.com
```
Comment (martinpeck) [inline config.sample.yaml:33]: suggest `example.com` as your example as `company.com` is a real domain

Comment (martinpeck) [reply config.sample.yaml:33]: TBH, should it simply be `__CHANGE_ME__` to avoid this accidentally working?

Comment (t-young31) [inline config.sample.yaml:33]:
```suggestion
    email: __CHANGE_ME__  # e.g. employee@example.com
```

Comment (martinpeck) [reply config.sample.yaml:33]: works for me!

Comment (martinpeck) [reply config.sample.yaml:33]: want me to commit that suggestion, or you?

**Review by martinpeck — APPROVED** [review]
LGTM

**Review by damoodamoo — APPROVED** [review]
awesome!

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4541072075 (with refid `7014b722`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/210#issuecomment-1486453731) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4542199374 (with refid `7014b722`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/210#issuecomment-1486635570) from @t-young31)

**Comment by t-young31** [comment]
/test-force-approve

destroy failed with known bug. Resolved by #28, if it ever runs

**Pull request merged**